Anomali Match - Added support to a list of *job_id* in the command **… (#11971)
demisto/content: Packs/Anomali_Enterprise/ReleaseNotes/1_0_2.md

#### Integrations
##### Anomali Match

> - Added support to a list of *job_id* in the command ***anomali-enterprise-retro-forensic-search-results***.

Reviewer: ```suggestion
- The *job_id* argument for the ***anomali-enterprise-retro-forensic-search-results*** command now supports a list of IDs.
```
